### ISSUE TITLE
Missing Declaration Files in NPM package 1.0.5 (*.d.ts files)

### ISSUE THREAD

**Issue by bugbytesinc** [hashgraph/hedera-wallet-connect#46]
Receiving the following feedback in development UI:

```
Could not find a declaration file for module '@hashgraph/hedera-wallet-connect'. '/home/[sic]/node_modules/.pnpm/@hashgraph+hedera-wallet-connect@1.0.5_typescript@5.3.3/node_modules/@hashgraph/hedera-wallet-connect/dist/node-cjs.js' implicitly has an 'any' type.
```

The distribution package.json file has the following entry:

```
"types": "./dist/types/index.d.ts",
```

But they are missing in the downloaded npmjs package:

![image](https://github.com/hashgraph/hedera-wallet-connect/assets/4412843/bbe206cd-6fd9-4f50-9c6e-05fac380e94b)

**Comment by hgraphql** [comment]
Hi @bugbytesinc this should be fixed with v1.0.6 https://www.npmjs.com/package/@hashgraph/hedera-wallet-connect/v/1.0.6 which was just published this morning

Pluto had raised an issue here as well - https://github.com/hashgraph/hedera-wallet-connect/issues/41

Let me know if you still have issues with this in v1.0.6

**Comment by hgraphql** [comment]
@bugbytesinc closing this issue as its fixed in v1.0.6, please let me know if you experience additional issues with this
